crates/serpen/tests/fixtures/ast_rewriting/alias_transformation_test/main.py

```
@@ -0,0 +1,64 @@
+
+This script demonstrates the issue where import aliases are not being transformed properly.
+Without the transformer logic, the bundled output will contain both:
+1. Alias assignments (generated by generate_alias_assignments)
+2. Original import statements (that should be removed/transformed)
+
+This creates redundant code and potentially confusing output.
+"""
+
+# These regular import aliases should be transformed/removed in bundled output
+import json as j
+import os as operating_system
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)